### DISCUSSION TITLE
Button focus visiblility

### DISCUSSION THREAD

**Discussion by RonLinu** [SpiralLinux/SpiralLinux-project#39] [General]
Hi,

Congratulations for Spiral Linux version. It compared favorably with Mint Debian Edition and other mainstream distros. It is on course to be my daily driver :)

About this discussion: is it possible to make a button in focus to stand out more visibly? I tried several color themes without any improvements.

As an example, in the Session window shown below, the Restart button is in focus. But we can hardly see that it is the case, just very very tiny vertical lines on the button. With the mouse, it is not a problem as we click the desired button but with the keyboard using the TAB key or arrow keys, it is easy to activate the wrong button.

![Screenshot from 2022-07-04 19-35-03](https://user-images.githubusercontent.com/108692042/177225584-823d8d64-6120-42f4-a2fc-f5226e92cfe2.png)

Ron

**Comment by geckolinux** [comment]
Hi there, thanks a lot for posting your experience, I'm really glad that it's working well for you.

You're absolutely right that the button focus isn't ideal with Numix. I mainly chose Numix because it has the best folor color contrast of all the themes available in the Debian repos, but I've run into the same issue with the dialog box that you show here.

You might want to try to a theme based on Yaru such as Yaru Colors:
https://github.com/Jannomag/Yaru-Colors/releases
It also has good font color contrast and I believe it has better button focus contrast.
If you plan on using Flatpaks you'll also want to install the same theme via Flatpak that you chose for the main system (for example **Yaru-red** in the Cinnamon appearance settings and then install the `org.gtk.Gtk3theme.Yaru-Red` Flatpak).

**Reply by RonLinu** [reply]
Thanks for the suggestion. I'll check this out.
I never downloaded new themes beside the ones already available in my distros so I have something new to learn. 
Part of the fun!

**Reply by geckolinux** [reply]
It's fairly easy, if you install it from a compressed archive just place the folder with the name of the theme (for example `Yaru-red`) in `/usr/share/themes/` to install it for all users (which requires root privileges, but SpiralLinux makes it easy with a right-click action in the file manager to open a folder as root) or else you can install it for your personal user account in the `.themes/` directory (hidden by default).
